tools/devops-tasks/ApiCompat/readme.md

```diff
@@ -0,0 +1,80 @@
+
+# ApiCompat task
+
+API Compat tool compares two sets of assemblies and determines if the implementation has incompatibilities with the contract.
+
+This is an Azure Pipelines encapsulation of the ApiCompat tool. To get the lastest version of this tool please visit [dotnet/arcade](https://github.com/dotnet/arcade/tree/master/src/Microsoft.DotNet.ApiCompat) repo.
+
+## Configuration
+
+### Contracts root folder
+
+Path to the folder that contains the assemblies to compare *against*.
+
+### Contracts files name
+
+Space separated list of the names of the assemblies to compare to.
+
+### Implementation assemblies folder
+
+Folder that contains the assemblies that are going to be compared against the contracts.
+
+### Fails on Issue
+
+Checking this will cause the build process to fail when ApiCompat finds one or more incompatibilities. Leaving it unchecked will flag the task as *successful with issues*. For Pull Requests we recommend to leave this checked to alert to alert the Pull Request user that his implementations caused incompatibilities.
```

Comment:
```suggestion
Checking this will cause the build process to fail when ApiCompat finds one or more incompatibilities. Leaving it unchecked will flag the task as *successful with issues*. For Pull Requests we recommend leaving this checked to alert the Pull Request user that his implementations caused incompatibilities.
```